packages/lesswrong/server/userJobAdCron.tsx

```diff
@@ -62,12 +65,14 @@ const sendJobAdReminderEmails = async () => {
   logger(`Sent email reminders for ${jobNames.join(', ')} to ${users.length} users`)
 }
 
-addCronJob({
-  name: 'sendJobAdReminderEmails',
-  interval: `every 1 day`,
-  job() {
-    void sendJobAdReminderEmails();
-  }
-});
+if (isEAForum) {
+  addCronJob({
+    name: 'sendJobAdReminderEmails',
+    interval: `every 1 day`,
+    job() {
+      void sendJobAdReminderEmails();
+    }
+  });
+}
```

Comment:
Forgot to forum-gate this, oops. Didn't cause any issues though since no other sites have job ads.